hw03_andersonz.py

```diff
@@ -0,0 +1,167 @@
+# Google Doc Link:https://docs.google.com/document/d/16mN4UUvcI00iAaC6fagV-eFSV3rITOZuQpNa9O2vbgg/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+#
+#
+#################################################################################
+
+import turtle
+
+
+def draw_triangle(bob):
+    """
+Draws a triangle
+   """
+
+    bob.pencolor("black")
+    bob.pensize(15)
+    bob.forward(200)
+    bob.left(130)
+    bob.forward(311.145)
+    bob.left(100)
+    bob.forward(311.145)
+    bob.left(130)
+    bob.forward(200)
+
+
+# ....
+
+
+def white_line(bob):
+    """
+Connects white line to triangle
+    """
+    bob.pencolor("white")
+    bob.penup()
+    bob.forward(-200)
+    bob.left(90)
+    bob.forward(130)
+    bob.right(90)
+    bob.forward(-200)
+    bob.pensize(10)
+    bob.pendown()
+    bob.forward(313)
+
+    # ...
+
+
+def red_line(bob):
+    """
+    Connects red line to triangle
+        """
+    bob.pencolor("red")
+    bob.penup()
+    bob.goto(-87.00, 135.00)
+    bob.pendown()
+    for i in range(10):
+        bob.pensize(i + 5)
+        bob.forward(20)
+        bob.left(.7)
+    for i in range(10, 20):
+        bob.pensize(i + 5)
+        bob.forward(10)
+    bob.forward(150)
+
+
+def yellow_line(bob):
+    """
+    Connects yellow line to triangle
+        """
+    bob.pencolor("yellow")
+    bob.penup()
+    bob.goto(-87.00, 132.500)
+    bob.setheading(0)
+    bob.pendown()
+    for i in range(10):
+        bob.pensize(i + 5)
+        bob.forward(20)
+        bob.left(.4)
+    for i in range(10, 20):
+        bob.pensize(i + 5)
+        bob.forward(10)
+    bob.forward(150)
+
+
+def green_line(bob):
+    """
+    Connects green line to triangle
+        """
+    bob.pencolor("green")
+    bob.penup()
+    bob.goto(-87.00, 130.00)
+    bob.setheading(0)
+    bob.pendown()
+    for i in range(10):
+        bob.pensize(i + 5)
+        bob.forward(20)
+    for i in range(10, 20):
+        bob.pensize(i + 5)
+        bob.forward(10)
+    bob.forward(150)
+
+
+def blue_line(bob):
+    """
+    Connects blue line to triangle
+        """
+    bob.pencolor("blue")
+    bob.penup()
+    bob.goto(-87.00, 127.500)
+    bob.setheading(0)
+    bob.pendown()
+    for i in range(10):
+        bob.pensize(i + 5)
+        bob.forward(20)
+        bob.left(-.4)
+    for i in range(10, 20):
+        bob.pensize(i + 5)
+        bob.forward(10)
+    bob.forward(150)
+
+
+def purple_line(bob):
+    """
+    Connects purple line to triangle
+        """
+    bob.pencolor("purple")
+    bob.penup()
+    bob.goto(-87.00, 125.00)
+    bob.setheading(0)
+    bob.pendown()
+    for i in range(10):
+        bob.pensize(i + 5)
+        bob.forward(20)
+        bob.left(-.8)
+    for i in range(10, 20):
+        bob.pensize(i + 5)
+        bob.forward(10)
+    bob.forward(150)
+
+
+def main():
+    """
+    Docstring for main. Should describe the main functionality of this file.

```

Comment:
Replace this with a meaningful docstring (should describe the purpose of this program).